src/app/pages/signing/disclaimer-button/disclaimer-button.component.html

```diff
@@ -0,0 +1,13 @@
+<a class="btn btn-danger disclaimer-button" data-toggle="popover" data-trigger="focus" tabindex="0" role="button"
+   title="Your Photo Links. Your Responsibility." data-html="true"
+   data-content="<p><strong>Note: If you decide to embed photos into your message, you agree
+                    that, since Virtrolio does not store these images, you will take full responsibility
+                    for whether the photos will work now and in the future, whether for you or for others.</strong></p>
+                    <p>For example, Google Photos is inconsistent in whether its embed links can be viewed by other
+                    users (putting the photos into a photo album is recommended if you are trying this approach). Other
+                    services such as Google Drive or Imgur or Pinterest can be used. Virtrolio is not endorsing any
+                    services mentioned here, merely making you aware of options.</p>
+                    <p>See our <a href='/terms-of-service' target='_blank'>Terms of Service</a> and
+                    <a href='/faq' target='_blank'>FAQ</a> for more information.</p>">
```

Comment:
Same suggestion as above:
```suggestion
   data-content="<p><strong>Note: Virtrolio does not store any of the photos you embed into your message! If you decide to embed photos into your message, you agree
                    that you will take full responsibility
                    for whether the link to your photos will be valid now and in the future, whether for you or for the person you are sending to.</strong></p>
                    <p>For example, Google Photos is inconsistent in whether its embed links can be viewed by other
                    users (putting the photos into a shared public photo album is recommended if you are trying this approach). Other
                    services such as Google Drive or Imgur or Pinterest tend to be more reliable. Virtrolio is not endorsing any
                    services mentioned here, merely making you aware of your options.</p>
                    <p>See our <a href='/terms-of-service' target='_blank'>Terms of Service</a> and
                    <a href='/faq' target='_blank'>FAQ</a> for more information.</p>">
```